apps/andi/assets/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Can we do this at the top level .gitignore? If not, what is the reason we need to be under `andi/assets`?

Reply:
If we can't do this I'd at least appreciate an explanation.